### ISSUE TITLE
logback.xml should not be packaged within the jar

### ISSUE THREAD

**Issue by elenigen** [nguyenq/tess4j#38]
At the startup of the application, if we add our own logback.xml inside a new project, we can see this warning:

``` bash
|-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs multiple times on the classpath.
|-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs at [jar:file:/Users/[user]/.m2/repository/net/sourceforge/tess4j/tess4j/3.2.1/tess4j-3.2.1.jar!/logback.xml]
|-WARN in ch.qos.logback.classic.LoggerContext[default] - Resource [logback.xml] occurs at [file:/[project]/target/classes/logback.xml]
```

The result is pretty bad, since we lose control on our logging configurations, there's a workaroud, we can specify a distinct name for the config file etc., but it's not supposed to be like that.

In the pom.xml, it easy to exclude the file when packaging the jar:

``` xml
                <plugin>
                    <groupId>org.apache.maven.plugins</groupId>
                    <artifactId>maven-jar-plugin</artifactId>
                    <configuration>
                        <excludes>
                            <exclude>**/logback.xml</exclude>
                        </excludes>
                    </configuration>
                </plugin>
```

**Comment by 4F2E4A2E** [comment]
Thank you for creating this issue, as soon as it is confirmed to be error prune after passing integration tests or in a productiv environment we will make this change happen.

Please remove the logback file from the jar and run your tests in your application.

**Comment by 4F2E4A2E** [comment]
> Please remove the logback file from the jar and run your tests in your application.

did you have tried it yet? Can you confirm it?

**Comment by elenigen** [comment]
I'm using it in a small personal project, so I don't even have tests, but you can see other projects also had this issue:
https://github.com/playframework/playframework/issues/3245
It's just not supposed to be package within libraries.
Thanks for the feedback!

**Comment by 4F2E4A2E** [comment]
This is a very project/magician point of view of it. It makes totally sense untill it does not. 
This is not the way i like to make decisions, but hey i will get into it know.
Thanks

**Comment by phillipjohnson** [comment]
Any update on this? Libraries should be logging to a facade allow end-users to provide their own logging implementation. By including `logback.xml` in a packaged jar, you are controlling the logs of users of tess4j.

In case anyone else is looking for a workaround, the easiest I found is to pass this system property on JVM startup: `-Dlogging.config=src/main/resources/logback-spring.xml`. Obviously you would point to your own configuration file. If you are using spring boot, it can be as simple as:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<configuration>
    <include resource="org/springframework/boot/logging/logback/base.xml"/>
</configuration>
```

**Comment by nguyenq** [comment]
A fix has been committed for this: c6e5687. It'll be in the next release.

**Comment by nguyenq** [comment]
Fixed in [tess4j-3.4.4](https://github.com/nguyenq/tess4j/releases/tag/tess4j-3.4.4) release.
